package.json

```
@@ -1,87 +1,89 @@
 {
-    "name": "freecodecamp-news-theme",
-    "description": "The Ghost theme for https://www.freecodecamp.org/news",
-    "demo": "https://www.freecodecamp.org/news",
-    "version": "2.9.0",
-    "engines": {
-        "ghost": ">=2.0.0",
-        "ghost-api": "v2"
-    },
-    "license": "MIT",
-    "scripts": {
-        "develop": "gulp",
-        "dev": "npm run develop",
-        "zip": "gulp zip",
-        "test": "gscan .",
-        "test:ci": "gscan --fatal --verbose .",
-        "start-ghost": "cd ../../.. && ghost start"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/freeCodeCamp/news-theme.git"
-    },
-    "bugs": "https://github.com/freeCodeCamp/news-theme/issues",
-    "contributors": [
-        "https://github.com/freeCodeCamp/news-theme/graphs/contributors"
-    ],
-    "author": {
-        "email": "team@freecodecamp.org",
-        "name": "freeCodeCamp.org",
-        "url": "https://www.freecodecamp.org"
-    },
-    "keywords": [
-        "ghost-theme"
-    ],
-    "devDependencies": {
-        "@freecodecamp/eslint-config": "^2.0.2",
-        "autoprefixer": "^9.7.6",
-        "babel-eslint": "^10.1.0",
-        "cssnano": "^4.1.10",
-        "eslint": "^6.8.0",
-        "eslint-config-prettier": "^6.10.1",
-        "eslint-plugin-import": "^2.20.2",
-        "eslint-plugin-prefer-object-spread": "^1.2.1",
-        "eslint-plugin-prettier": "^3.1.2",
-        "eslint-plugin-react": "^7.19.0",
-        "gscan": "^3.5.2",
-        "gulp": "^4.0.2",
-        "gulp-filter": "^6.0.0",
-        "gulp-livereload": "^4.0.2",
-        "gulp-postcss": "^8.0.0",
-        "gulp-sourcemaps": "^2.6.5",
-        "gulp-uglify": "^3.0.2",
-        "gulp-util": "^3.0.8",
-        "gulp-watch": "^5.0.1",
-        "gulp-zip": "^5.0.1",
-        "postcss-color-function": "^4.1.0",
-        "postcss-custom-properties": "^9.1.1",
-        "postcss-easy-import": "^3.0.0",
-        "prettier": "^2.0.4"
-    },
-    "browserslist": [
-        "defaults"
-    ],
-    "config": {
-        "posts_per_page": 25,
-        "image_sizes": {
-            "xxs": {
-                "width": 30
-            },
-            "xs": {
-                "width": 100
-            },
-            "s": {
-                "width": 300
-            },
-            "m": {
-                "width": 600
-            },
-            "l": {
-                "width": 1000
-            },
-            "xl": {
-                "width": 2000
-            }
-        }
+  "name": "freecodecamp-news-theme",
+  "description": "The Ghost theme for https://www.freecodecamp.org/news",
+  "demo": "https://www.freecodecamp.org/news",
+  "version": "2.9.0",
+  "engines": {
+    "ghost": ">=2.0.0",
+    "ghost-api": "v2"
```

```suggestion
    "ghost": ">=4.0.0",
    "ghost-api": "v4"
```

Is there a reason to leave these locked at v2 instead of the current versions? If I update them in my local build, the warning about deprecated API version goes away.
```console

Your theme has 1 warning!
------

Warnings
--------
- Warning: The {{@site.lang}} helper should be replaced with {{@site.locale}}
Files: default.hbs,error-404.hbs,error.hbs,partials/comments.hbs,partials/social-row.hbs,search-results.hbs
```

---

Yes - that is correct. I opened this PR before I was done with it and hence blocked. Will update the PR to address the warnings.